addons/api/tests/unit/serializers/target-test.js

```diff
@@ -85,6 +87,7 @@ module('Unit | Serializer | target', function (hooks) {
       enable_session_recording: false,
       storage_bucket_id: null,
       address: '0.0.0.0',
+      with_aliases: [],
```

Comment:
Why are we passing `with_aliases` empty?

Reply:
since I made a test with non empty array in previous test case, I didn't do here and other places, but since this is SSH targets, it makes sense to do a non empty test case? I'll pass a non empty arr